nconnect.go

```diff
@@ -209,18 +217,17 @@ func NewNconnect(opts *config.Opts) (*nconnect, error) {
 	}
 
 	tsConfig := &ts.Config{
-		TunaMaxPrice:                 opts.TunaMaxPrice,
-		TunaMinNanoPayFee:            opts.TunaMinFee,
-		TunaNanoPayFeeRatio:          opts.TunaFeeRatio,
-		TunaIPFilter:                 &geo.IPFilter{Allow: allowedIP, Disallow: disallowedIP},
-		TunaNknFilter:                &filter.NknFilter{Allow: allowedNknAddrs, Disallow: disallowedNknAddrs},
-		TunaServiceName:              opts.TunaServiceName,
-		TunaDownloadGeoDB:            !opts.TunaDisableDownloadGeoDB,
-		TunaGeoDBPath:                opts.TunaGeoDBPath,
-		TunaMeasureBandwidth:         !opts.TunaDisableMeasureBandwidth,
-		TunaMeasureStoragePath:       opts.TunaMeasureStoragePath,
-		TunaMeasurementBytesDownLink: opts.TunaMeasureBandwidthBytes,
```

Comment:
Why remove this?

Reply:
I didn't modify this section. Maybe keyboard mistake. let me recover it.